release notes/v0.43.0.md

```diff
@@ -0,0 +1,124 @@
+k6 v0.43.0 is here! :tada:
+
+Notable changes in this release include:
+- [xk6-browser](https://github.com/grafana/xk6-browser) is now bundled in k6 as an experimental module, and usable without a separate binary or compilation step!
+- Native support for JavaScript's `async`/`await`.
+- A new experimental module for distributed tracing.
+- Large refactoring of core components to simplify the code base, improve maintainability, and increase test coverage.
+- Bug fixes, UX improvements, and maintenance.
+
+Keep reading for the details.
+
+## Breaking changes
+
+- [#2807](https://github.com/grafana/k6/pull/2807) Use non-zero exit codes for tests aborted by Ctrl+C or the REST API.
+
+  Aborting a test run with Ctrl+C will now exit with code `105`, and stopping via the REST API will exit with code `103`.
+
+
+## New Features
+
+### xk6-browser is now a built-in module [#2884](https://github.com/grafana/k6/pull/2884)
+
+[xk6-browser](https://github.com/grafana/xk6-browser), a k6 extension that enables browser automation and end-to-end testing from k6 scripts, now comes bundled with k6 under the `k6/experimental/browser` import path. This makes it more accessible, as users no longer need to build their own binary with [xk6](https://github.com/grafana/xk6), or use a separate binary to run browser tests.
```

Comment:
Can we just do the `that enables`->`for` change, but otherwise keep the xk6-browser definition in the same sentence? It reads more naturally to me, and doesn't add a pause and another paragraph, since we'd prefer keeping this document as short as possible.

Reply:
In this case I'm not sure. The original structure of the first sentence has a pretty lengthy "middle branch" that separates the subject from the verb. This creates a lot for the reader to hold in the head. Even though the other sentence might have a better flow, I prefer maximum directness, even if it comes off a bit brutalistic in aesthetic.

Since it's background information, I'd be fine with just cutting it:

> This release upgrades xk6-browser from an extension to an experimental module. In other words, you can now use the standard k6 application for browser and end-to-end-testing―no need to build a custom binary.

If the background info seems necessary, I think it's okay to separate the two paragraphs, since they represent separate ideas: background info, new info. This doesn't  increase the content of the text, only the length of vertical scroll. But short paragraphs increase readability, so I'd say it's a wash.

Reply:
Alright. I changed your suggestion slightly in 2acbcbc9. Let me know.